hypervisor/qemu/qemu_arm64.go

```diff
@@ -0,0 +1,59 @@
+// +build linux,arm64
+
+package qemu
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hyperhq/runv/hypervisor"
+)
+
+const (
+	QEMU_SYSTEM_EXE    = "qemu-system-aarch64"
+	VM_MIN_MEMORY_SIZE = 128
+)
+
+func (qc *QemuContext) arguments(ctx *hypervisor.VmContext) []string {
+	if ctx.Boot == nil {
+		ctx.Boot = &hypervisor.BootConfig{
+			CPU:    1,
+			Memory: VM_MIN_MEMORY_SIZE,
+			Kernel: hypervisor.DefaultKernel,
+			Initrd: hypervisor.DefaultInitrd,
+		}
+	}
+	boot := ctx.Boot
+	qc.cpus = boot.CPU
+
+	// Currently the default memory size is fixed to 128 MiB.
+	if boot.Memory < VM_MIN_MEMORY_SIZE {
+		boot.Memory = VM_MIN_MEMORY_SIZE
+	}
+
+	var memParams, cpuParams string
+	if boot.HotAddCpuMem {
+		memParams = fmt.Sprintf("size=%d,slots=1,maxmem=%dM", boot.Memory, hypervisor.DefaultMaxMem)
+		cpuParams = fmt.Sprintf("cpus=%d,maxcpus=%d", boot.CPU, hypervisor.DefaultMaxCpus)
+	} else {
+		memParams = strconv.Itoa(boot.Memory)
+		cpuParams = strconv.Itoa(boot.CPU)
+	}
+
+	return []string{
+		"-machine", "virt,usb=off", "-cpu", "cortex-a57",
```

Comment:
for those have `/dev/kvm`, this line should be 

```		"-machine", "virt,accel=kvm,usb=off", "-cpu", "host",```

For example, my debian on pine64